index.js

```diff
@@ -0,0 +1,130 @@
+inputBox.onkeyup = () => {
+    userEnteredTask = inputBox.value;
+    if (userEnteredTask.trim() != 0) {
+        addButton.classList.add("active");
+    }
+    else {
+        addButton.classList.remove("active");
+    }
+}
+
+
+addButton.onclick = () => {
+    taskListArray.push(userEnteredTask);
+    localStorage.setItem("Task list", JSON.stringify(taskListArray));
+    showListOfTasks();
+    addButton.classList.remove("active");
+}
+
+function showListOfTasks() {
+
+    if (taskListArray.length > 0 || completeTaskList.length > 0) {
+        deleteAllButton.classList.add("active");
+    } else {
+        deleteAllButton.classList.remove("active");
+    }
+
+    let newLiTag = "";
+    if (taskListArray.length == 0) {
+        toDoList.innerHTML = newLiTag;
+    }
+    taskListArray.forEach((element, index) => {
+        newLiTag += `<li><div class="complete-task-icon" onclick="completeTask(${index})"><i class="fa fa-check-square fa-border" aria-hidden="true"></i></div>  <div class="element-container">${element}</div><div class="side-icons"><div><button class="edit-icon" onclick="editTask(${index} )">Edit</button></div><div class="delete-icon" onclick="deleteTask(${index})" ><i class="fas fa-trash"></i></div></div></li>`;
+        toDoList.innerHTML = newLiTag;
+        inputBox.value = "";
+
+    });
+    pendingTask.innerHTML = `You have ${taskListArray.length} pending task.`
+}
+
+function editTask(index) {
+    addButton.style.display = "none";
+    saveButton.style.display = "block";
+    updateindex = index;
+    inputBox.value = taskListArray[index];
+
+}
+
+
+saveButton.onclick = () => {
+    addButton.style.display = "block";
+    addButton.classList.remove("active");
+    saveButton.style.display = "none";
+
+    taskListArray[updateindex] = inputBox.value;
+    localStorage.setItem("Task list", JSON.stringify(taskListArray));
+    showListOfTasks();
+
+}
+
+function deleteTask(index) {
+    taskListArray.splice(index, 1);
+    localStorage.setItem("Task list", JSON.stringify(taskListArray));
+    showListOfTasks();
+}
+
+
+
+function completeTask(index) {
+    completeTaskList.push(taskListArray[index]);
+    localStorage.setItem("Completed task", JSON.stringify(completeTaskList));
+    showCompletedTask();
+    deleteTask(index);
+
+}
+
+function showCompletedTask() {
+    if (completeTaskList.length == 0) {
+        completedTaskList.innerHTML = "";
+    }
+
+    newLiTag = `<h1>Completed Task</h1>`;
+    completeTaskList.forEach((element, index) => {
+        newLiTag += `<li><div class="element-container">${element}</div><div class="complete-list-delete-icon" onclick="deleteCompletedTask(${index})" ><i class="fas fa-trash"></i></div></li>`;
+        completedTaskList.innerHTML = newLiTag;
+    });
+}
+
+function deleteCompletedTask(index) {
+    completeTaskList.splice(index, 1);
+    localStorage.setItem("Completed task", JSON.stringify(completeTaskList));
+    showCompletedTask();
+}
+
+
+deleteAllButton.onclick = () => {
```

Comment:
Same here.
Please define this function separately and then assign it to onclick property.